pom.xml

```diff
@@ -479,7 +479,7 @@
 		<dependency>
 			<groupId>com.google.gwt.inject</groupId>
 			<artifactId>gin</artifactId>
-			<version>${ginVersion}</version>
+			<version>1.5.0</version>
```

Comment:
We should be using gin version 2.1.2, not 1.5.0.  Was this version change intentional?